src/formpack/schema/fields.py

```diff
@@ -452,6 +469,41 @@ def get_disaggregated_stats(self, metrics, top_splitters,
 
         return stats
 
+    def format(self, val, xls_types=False, *args, **kwargs):
+        if val is None:
+            val = ''
+
+        if not xls_types:
+            return {self.name: val}
+
+        _date = val
+        try:
+            _date = parser.parse(val)
+        except ValueError:
+            pass
+        if isinstance(_date, datetime):
```

Comment:
Same comment about `else`? 

if `parser.parse` works as expected, it returns a `datetime` object. 

```
        try:
            _date = parser.parse(val)
        except ValueError:
            pass
        else:
             # This assertion should be always True
             # assert type(_date) == datetime
             _date = _date.date()
                   
```        

Did I miss something?

Reply:
You missed nothing 💁